README.md

```diff
@@ -4,3 +4,64 @@
 Churros
 ====
 This is a humble library dedicated to making the life of developers easier.
+
+Installation
+====
+Be sure to have JCenter in the base gradle file.
+```groovy
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+```
+
+Then it's a matter of adding the dependency.
+```groovy
+dependencies {
+    // Churros
+    implementation "com.asapp.churros:churros:0.1.0"
+}
+```
+
+Things we have
+====
+* runIf
```

Comment:
How about having each available function being an H2, so that we can more easily link to them?

```suggestion
## runIf
```